COMMITTERS.adoc

```diff
@@ -0,0 +1,164 @@
+
+== Issues Fixed
+
+When a PR fixes some issues, it's good practice to add it in the description.
+
+One issue per line with something like:
+
+[source,asciidoc]
+----
+* Fix #444
+* Fix #555
+----
+
+Note that GitHub won't detect issues properly if you do something like
+`Fix #444 #555`.
+
+== Affecting Labels and Milestones
+
+Affecting labels and milestones is very important in our process.
+
+Before each release, we check that we don't have any orphan i.e.
+closed issues that don't have either a milestone affected or
+some of the "excluding" labels.
+
+Thus:
+
+* when your merge a pull request, please affect it with the current milestone marked with "master".
+* if the pull request has issues associated (i.e. if the pull request fixes some issues),
+  also affect the milestone to the issues.
+* if you close a pull request because we won't merge it, please add the appropriate `triage/` label:
+  `triage/invalid`, `triage/out-of-date`, `triage/wontfix` are usually in order.
+* if you close an issue because it has been fixed, either add the milestone
+  if you can find it easily or use the `triage/out-of-date` if you can't.
+* if you close an issue for any other reason, one of the aforementioned `triage/` labels
+  is probably adequate.
+
+== Backporting Process
+
+When we release a new version of Quarkus, we usually do a bugfix
+release a couple of weeks after.
+
+Everytime, we do a release, we create a release branch to host
+the commits for these bugfix releases.
+
+All the pull requests are merged in master so for the new feature 
+release of Quarkus.
+They won't be integrated in the previous version branch.
+
+One of your pull request might qualify for being backported to this
+bugfix branch.
+
+Good examples of that might be:
+
+* bugfixes
+* documentation fixes
+* usability fixes
+
+Obviously, the barrier is higher for large pull requests as
+they might be more risky to backport.
+But sometimes, we just have to backport them anyway.
+
+If you think your pull request or the pull request you review might be a good backport candidate,
+please add the `triage/backport?` label.
+
+The question mark is important:
+it is not automatic and we carefully review each pull request before backporting.
+
+Thus, if not entirely clear, don't hesitate to add a comment to the pull request
+when adding the label.
+
+And don't be surprised if we come to you with some questions about it
+when we prepare the next bugfix release.
+
+== Good First Issues
+
+We need to find the right balance between fixing the issue right away
+or try to get a new contributor onboard.
+
+It's not always easy to find it but if you think the issue is appropriate,
+affecting it the `good first issue` label for some time might be a good thing.
+
+Obviously, critical bugs are not good candidates :).
+
+== I Did Something Wrong, What Should I Do?
+
+Take a deep breath and don't worry, it happens.
+
+Just ping @gsmet and @geoand and we will find a solution.
```

Comment:
Maybe suggest asking in Zulip?

Reply:
yeah I wasn't sure what was the best way. But sure I can suggest that.